log/adapter.go

```diff
@@ -0,0 +1,103 @@
+package log
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/ydb-platform/ydb-go-sdk/v3/logs"
+)
+
+type adapter struct {
+	l Logger
+}
+
+func (a adapter) Log(opts logs.Options, msg string, fields ...logs.Field) {
+	logger := a.l
+	for _, name := range opts.Scope {
+		logger = logger.WithName(name)
+	}
+
+	var f func(string, ...interface{})
+	switch opts.Lvl {
+	case logs.TRACE:
+		f = logger.Tracef
+	case logs.DEBUG:
+		f = logger.Debugf
+	case logs.INFO:
+		f = logger.Infof
+	case logs.WARN:
+		f = logger.Warnf
+	case logs.ERROR:
+		f = logger.Errorf
+	case logs.FATAL:
+		f = logger.Fatalf
+	default:
+		return
```

Comment:
Need explicit return for quiet level and some error for unknown level as default

Reply:
And need test for all levels